src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -559,7 +559,7 @@ public void SetWindowPosition_Unix_ThrowsPlatformNotSupportedException()
         Assert.Throws<PlatformNotSupportedException>(() => Console.SetWindowPosition(50, 50));
     }
 
-    [PlatformSpecific((TestPlatforms.Windows) | (TestPlatforms.AnyUnix & ~TestPlatforms.Browser & ~TestPlatforms.iOS & ~TestPlatforms.MacCatalyst & ~TestPlatforms.tvOS))]
+    [PlatformSpecific((TestPlatforms.Windows)]
```

Comment:
I've updated this to be skipped on Unix.

When I run it, I get no error from the kernel when setting these sizes.
And even if there were an error, `Console.Unix` has no code to turn it into the `ArgumentOutOfRangeException` this test expects.

On Windows, does calling `Console.SetWindowSize` resizes the visual window?
If it does, I'm not seeing this with Gnome terminal.
I think the `TIOCSWINSZ` may be meant to inform the kernel about the actual size, instead of requesting a different one.

Reply:
These comments are not related to the issue being fixed here.

Unless we know `Console.SetWindowSize` works as expected on Unix/Linux, we can create a new issue to look into this further.

Reply:
> we can create a new issue to look into this further.

I've created https://github.com/dotnet/runtime/issues/96208 for this.